### PR TITLE
vk.com provider registered

### PR DESCRIPTION
**Pull request by nbolatov**

### CODE REVIEW

File: src/authDataManager/index.js [hunk 1]
```diff
@@ -1,59 +1,62 @@
...
-let github = require("./github");
-let twitter = require("./twitter");
-let spotify = require("./spotify");
-let digits = require("./twitter"); // digits tokens are validated by twitter
-let janrainengage = require("./janrainengage");
-let janraincapture = require("./janraincapture");
-
-let anonymous = {
-  validateAuthData: () => {
+"use strict";
```
Comment (flovilmart) [inline src/authDataManager/index.js]: can you revert all the unnecessary changes?

**Comment by facebook-github-bot-2** [comment]
@nbolatov updated the pull request - [view changes](https://github.com/ParsePlatform/parse-server/pull/2579/files/25705d3d1272e44d5c5801050adf8eafec95cd27..7dec6297421045c149aa49e4b888a02c38b96f9e)

**Comment by facebook-github-bot-3** [comment]
@nbolatov updated the pull request - [view changes](https://github.com/ParsePlatform/parse-server/pull/2579/files/7dec6297421045c149aa49e4b888a02c38b96f9e..6e6b465eefcad5d97177989de335585cc37a898e)

**Comment by facebook-github-bot-7** [comment]
@nbolatov updated the pull request - [view changes](https://github.com/ParsePlatform/parse-server/pull/2579/files/6e6b465eefcad5d97177989de335585cc37a898e..891271dfd146f525a5bde0c41f1f9d75d05f6afc)

**Comment by flovilmart** [comment]
It needs a rebase on master :)

**Comment by nbolatov** [comment]
@flovilmart what should I do in order to pull successfully?

**Comment by flovilmart** [comment]
```
git fetch origin
git rebase origin/master
```

given that you're on your branch and that origin is this remote.

**Comment by facebook-github-bot-6** [comment]
@nbolatov updated the pull request - [view changes](https://github.com/ParsePlatform/parse-server/pull/2579/files/891271dfd146f525a5bde0c41f1f9d75d05f6afc..fe2992b1b3cbb5b427a5cb99e810a38a51207f9c)
